.go-version

```diff
@@ -1 +1 @@
-1.18.5
+1.19
```

Comment:
It may be good to also set a patch version?

Reply:
the `.0` release chops off the `.0` (we do the same thing!)